erica/application/FreischaltCode/FreischaltCode.py

```diff
@@ -1,19 +1,18 @@
-from datetime import date
-
 from erica.application.base_dto import BaseDto
+from erica.domain.FreischaltCode.FreischaltCode import FreischaltCodeRequestPayload, FreischaltCodeActivatePayload, \
+    FreischaltCodeRevocatePayload
 
 
 class FreischaltCodeRequestDto(BaseDto):
-    idnr: str
-    date_of_birth: date
+    payload: FreischaltCodeRequestPayload
+    clientIdentifier: str
 
 
-class FreischaltCodeRevocateDto(BaseDto):
-    idnr: str
-    elster_request_id: str
+class FreischaltCodeActivateDto(BaseDto):
+    payload: FreischaltCodeActivatePayload
+    clientIdentifier: str
 
 
-class FreischaltCodeActivateDto(BaseDto):
-    idnr: str
-    freischalt_code: str
-    elster_request_id: str
+class FreischaltCodeRevocateDto(BaseDto):
+    payload: FreischaltCodeRevocatePayload
```

Comment:
I suggest to have domain models and payload  and DTOs like this:
```
class FreischaltCode
  payload: FreischaltCodePayload
```

```
class FreischaltCodeDto
  payload: FreischaltCodePayloadDto
```

Reply:
I tried to change it accordingly. The problem I see is that we do not have a correct domain (because of all the legacy parts). Therefore, I find that middle ground a little shaky tbh.
Probably the Payload should not be the real domain model but probably the EricaRequest itself?